From 75e7922b3130e482eed8bd3f702b6071d230ea22 Mon Sep 17 00:00:00 2001
From: Luwei Zhou <b45643@freescale.com>
Date: Wed, 2 Apr 2014 09:15:16 +0800
Subject: [PATCH 0370/1074] ENGR00306134-2: ARM: dts: imx6sx-sdb: Add mag3110
 dts support

This patch add mag3110 sensor device tree support on i.MX6SX-SDB
platform.

Signed-off-by: Luwei Zhou <b45643@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sdb.dts |   10 ++++++++++
 1 files changed, 10 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sdb.dts b/arch/arm/boot/dts/imx6sx-sdb.dts
index 929f6c1..6d547f6 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb.dts
@@ -318,6 +318,15 @@
         pinctrl-names = "default";
         pinctrl-0 = <&pinctrl_i2c3_2>;
         status = "okay";
+
+	mag3110@0e {
+		compatible = "fsl,mag3110";
+		reg = <0x0e>;
+		position = <2>;
+		interrupt-parent = <&gpio6>;
+		interrupts = <5 1>;
+		shared-interrupt;
+	};
 };
 
 &i2c4 {
@@ -348,6 +357,7 @@
 				MX6SX_PAD_QSPI1B_DATA3__GPIO4_IO_27 0x17059
 				MX6SX_PAD_CSI_DATA04__GPIO1_IO_18 0x17059
 				MX6SX_PAD_CSI_DATA05__GPIO1_IO_19 0x17059
+				MX6SX_PAD_SD1_DATA3__GPIO6_IO_5 0xb000
 			>;
 		};
 	};
-- 
1.7.5.4

